trino-aws-proxy/src/test/java/io/trino/aws/proxy/server/TestOpaSecurity.java

```diff
@@ -81,11 +88,17 @@ public OpaMapper(OpaContainer container)
         }
 
         @Override
-        public OpaRequest toRequest(ParsedS3Request request, Optional<String> lowercaseAction, URI baseUri, Optional<Identity> identity)
+        public OpaRequestAdapter toRequest(ParsedS3Request request, Optional<String> lowercaseAction, URI baseUri, Optional<Identity> identity)
         {
             assertThat(identity).isPresent();
+            if (request.keyInBucket().equals("defaultAllowedTable")) {
```

Comment:
```suggestion
            if (request.keyInBucket().equals("default-allow")) {
```

This part of the code doesn't deal with tables